docs/data/toolpad/getting-started/overview.md

```diff
@@ -2,6 +2,8 @@
 
 <p class="description">Toolpad is a low-code UI builder for creating apps with MUI's suite of component libraries.</p>
 
+{{"component": "components/landing/Banner.js", "action": "tryDemo", "category": "ToolpadDocs", "description": "Check out Toolpad's interactive demo. No sign-up required!", "href": "https://demo.toolpad.io", "label": "Try Demo", "docs": true }}
```

Comment:
I'm not sure that we should use marketing pages components `{{"component": "components/landing/Banner.js"` inside the docs page. It looks like this now:

<img width="834" alt="Screenshot 2022-12-09 at 13 07 08" src="https://user-images.githubusercontent.com/3165635/206699131-c225bfe0-0991-4ca7-84ae-b399e7f5c5d2.png">

Maybe a banner would feel better like this for the docs:

<img width="809" alt="Screenshot 2022-12-09 at 13 07 43" src="https://user-images.githubusercontent.com/3165635/206699246-f394fbff-9f79-48cd-bdb5-d84eca5ead2f.png">